test/controllers/update/update.trusts.tell.us.about.it.controller.spec.ts

```diff
@@ -113,6 +113,15 @@ describe('Update - Trusts - Tell us about the trust', () => {
       expect(resp.text).toContain(UPDATE_TELL_US_ABOUT_TRUST_QUESTION);
     });
 
+    test('when feature flag is on and new trust no longer involved in the OE, re-render page with ceased date validation error', async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true);
+      mockGetApplicationData.mockReturnValue( { ...APPLICATION_DATA_MOCK } );
+      const resp = await request(app).post(UPDATE_TRUSTS_TELL_US_ABOUT_IT_URL).send({ stillInvolved: '0' });
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(TRUST_ENTER_CEASED_DATE);
```

Comment:
Could we also be checking whether the response contains `ERROR_LIST` to be consistent with the previous test?

Reply:
Done.